blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/BlueOceanWebURLBuilder.java

```diff
@@ -128,6 +129,8 @@ private BlueOceanWebURLBuilder() {
             if (blueResource instanceof BlueMultiBranchPipeline) {
                 return getOrgPrefix(item) + "/" + encodeURIComponent(((BluePipeline) blueResource).getFullName()) + "/branches";
             }
```

Comment:
This implementation will not work for orgs which are on a folder, bare in mind that most of the `ModifiableTopLevelItemGroup`s are also Items so they will come through this else if and never get the org ie. Folders. 
Maybe just getting the org and if not null return the org url instead of blue for all the cases and not only for `ModifiableTopLevelItemGroup`  that could be done a level above or checking all the if branches.